web/init/src/components/shared/RouteDecider.jsx

```diff
@@ -39,8 +39,8 @@ const ShipRoutesWrapper = ({ routes, headerEnabled, basePath }) => (
                   />}
                 />
               ))}
-              <Route exact path="/" component={() => <div className="flex1 flex-column justifyContent--center alignItems--center"><Loader size="60" /></div> } />
-              <Route exact path="/done" component={() =>  <StepDone />} />
+              <Route exact path={`${basePath}/`} component={() => <div className="flex1 flex-column justifyContent--center alignItems--center"><Loader size="60" /></div> } />
```

Comment:
is base path ever just `/`?

Reply:
`/` is defaulted if you omit `basePath`, I'll add additional documentation in the proptypes.

Reply:
Actually, I have a couple of doc tasks to add in a separate PR, will do that there

Reply:
ok cool, just wanted to make sure we didn't run into `//` and then the app crashes